contribute/design/1.x/csi/20190606-csi-volume-provisioning.md

```diff
@@ -312,6 +312,7 @@ metadata:
 spec:
   targetDeployment:
     spec:
+      monitoring: true
```

Comment:
Does this property help?
How about adding a container with monitoring image & command details?
In other words, the claim controller needs to bother about adding appropriate containers rather than understanding if its a monitoring or something else.

@vishnuitta @kmova Let us see if all kinds of defaults etc can be pushed into this structure?
It helps in avoiding writing these as functions / constants in code.